code/LINDA/LINDA_turf_tile.dm

```diff
@@ -207,7 +207,23 @@
 				if(excited_group)
 					last_share_check()
 
-	air.react()
+	if(air)  //How did this even break?
```

Comment:
Please do not do this. LINDA intentionally does not have null protection in many parts of it- This *should* runtime, because it should never happen.